docs/source/releases/v2.1.rst

```diff
@@ -15,12 +16,20 @@ Welcome to Oscar 2.1.
 Compatibility
 ~~~~~~~~~~~~~
 
+Oscar 2.1 is compatible with Django versions 2.2 and 3.0, and Python versions 3.5, 3.6 and 3.7.
```

Comment:
Is there any reason 3.8 is not listed here? Seems like the test suites are ok and I couldn't find an issue opened for a compatibility issue.

Reply:
No, an omission on my part. I'll add it.